packages/admin/cms-admin/src/common/MasterMenuRoutes.tsx

```diff
@@ -33,18 +34,24 @@ export function useRoutePropsFromMasterMenuData(items: MasterMenuData): RoutePro
 
 export interface MasterMenuRoutesProps {
     menu: MasterMenuData;
+    notFoundPage?: ReactNode;
```

Comment:
What do you think about using `fallback` instead? This could also be used to display an access denied page.

Reply:
Yes

Reply:
renamed the prop `notFoundPage` to `fallback`